test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/NuGet.PackageManagement.UI.Test.csproj

```diff
@@ -7,26 +7,25 @@
     <NETCoreWPFProject>true</NETCoreWPFProject>
     <TestProject>true</TestProject>
     <Description>Unit and integration tests for NuGet.PackageManagement.UI.</Description>
-    <GenerateBindingRedirectsOutputType>true</GenerateBindingRedirectsOutputType>
   </PropertyGroup>
 
   <ItemGroup>
+    <Reference Include="Microsoft.CSharp" />
+    <Reference Include="Microsoft.VisualStudio.Diagnostics.Assert, Version=16.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
+      <HintPath>$([MSBuild]::GetVsInstallRoot())\Common7\IDE\PrivateAssemblies\Microsoft.VisualStudio.Diagnostics.Assert.dll</HintPath>
```

Comment:
Why do we need this?
Isn't `PrivateAssemblies` already on the search path? 
We do that with: https://github.com/NuGet/NuGet.Client/blob/7e23737fdf89fe34c90ac8878c96843b2f0dd17e/src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/NuGet.PackageManagement.VisualStudio.csproj#L21-L23, which also comes from the same folder.

Reply:
This is an existing reference that I ordered amongst other references.  I'll see if this can be removed, but it's technically outside the scope of the PR, and I won't hold up merging for this.

Reply:
I missed that it was already there. Not relevant then. Your call.